shell/browser/native_window.cc

```diff
@@ -395,6 +395,14 @@ void NativeWindow::PreviewFile(const std::string& path,
 
 void NativeWindow::CloseFilePreview() {}
 
+gfx::Rect NativeWindow::GetWindowControlsOverlayRect() {
+  return overlay_rect_;
+}
+
+void NativeWindow::SetWindowControlsOverlayRect(const gfx::Rect& overlay_rect) {
```

Comment:
I don't see any code calling `SetWindowControlsOverlayRect` to set the overlay rect? (It is fine if it is needed by the Windows implementation.)

Reply:
@zcbenz yes - this is needed for the Windows implementation